scripts/Invoke-SasWorkstationE2E.py

```
@@ -0,0 +1,121 @@
+"workstation-malformed-agentswitchboard":("linux","malformed-agent-result","Plan",False,False,"FAIL"),
+"workstation-rollback-v2":("windows","rollback","Rollback",True,False,"PASS"),
+"workstation-unsupported-mac-v2":("unsupported","unsupported-platform","Plan",False,False,"UNSUPPORTED"),
+}
+
+
+def load(path:Path):return json.loads(path.read_text(encoding="utf-8-sig"))
+def write(path:Path,value):path.parent.mkdir(parents=True,exist_ok=True);path.write_text(json.dumps(value,indent=2)+"\n",encoding="utf-8")
+
+
+def validate(value,schema_path):
+    try:import jsonschema
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)